include/envoy/ssl/handshaker.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "envoy/network/connection.h"
+#include "envoy/network/post_io_action.h"
+
+#include "openssl/ssl.h"
+
+namespace Envoy {
+namespace Ssl {
+
+class HandshakeCallbacks {
+public:
+  virtual ~HandshakeCallbacks() = default;
+
+  /**
+   * @return the connection state.
+   */
+  virtual Network::Connection::State connectionState() const PURE;
+
+  virtual void onSuccess(SSL* ssl) PURE;
```

Comment:
comment a is required for each method in header

Reply:
Done.